composer.json

```diff
@@ -21,36 +21,37 @@
         }
     ],
     "require": {
-        "php": "^5.5.9 || ^7.0",
-        "paragonie/random_compat": "^1 || ^2 || ^9",
-        "symfony/form": "^2.8 || ^3.0 || ^4.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/security-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/templating": "^2.8 || ^3.0 || ^4.0",
-        "symfony/translation": "^2.8 || ^3.0 || ^4.0",
-        "symfony/twig-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/validator": "^2.8 || ^3.0 || ^4.0",
-        "twig/twig": "^1.28 || ^2.0"
+        "php": "^7.2",
+        "doctrine/persistence": "^1.3",
```

Comment:
technically, it is possible to use FOSUserBundle without `doctrine/persistence` (if you use a custom driver). So this should be handled with a conflict rule rather than a requirement.